### PR TITLE
New snapshot format blog post

### PR DESCRIPTION
**Pull request by snoyberg**

### CODE REVIEW

**Comment by juhp** [comment]
LGTM from a quick read

File: posts/new-snapshot-format.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+author: Michael Snoyman
+title: New Stackage snapshot format
+description: "We've been making some changes to how we build Stackage snapshots, and the file format used. This post covers those changes, and what's coming next."
+timestamp: 2019-07-04T13:00:00Z
+---
+
+I've been blogging off-and-on about [Pantry](https://www.stackage.org/package/pantry-tmp) and how it affects both Stack and Stackage. There have been a number of Github issues lying around about changes to how we'll build Stackage snapshots. Most of those changes have landed, and it's worth recapping the problems we were trying to solve, how things used to work, how they work now, and what's left.
```
Comment (alexeyzab) [inline posts/new-snapshot-format.md]: I think you can drop the hyphens in `off-and-on`:
```suggestion
I've been blogging off and on about [Pantry](https://www.stackage.org/package/pantry-tmp) and how it affects both Stack and Stackage. There have been a number of Github issues lying around about changes to how we'll build Stackage snapshots. Most of those changes have landed, and it's worth recapping the problems we were trying to solve, how things used to work, how they work now, and what's left.
```

File: posts/new-snapshot-format.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+* `stackage-curator` implements its own Cabal package building logic by working directly with Cabal-the-library. There's a lot of feature overlap with Stack, and different behavior in some corner cases.
+* The original Stackage file format used in `stackage-nightly` and `lts-haskell` is unique from the "custom snapshot" format that Stack supports, for only historical reasons.
+* The snapshot format is _big_. It contains a lot of information about module names, constraints, users, etc, that build tools don't need.
+* While there is cryptographic hash information on cabal files to deal with Hackage Revisions, there is no cryptographic hash information on the package tarball contents themselves.
+
+## Changes
+
+* We're scrapping the previous `stackage-curator` tool entirely.
+* There's a brand new `curator` tool, which uses the same snapshot format as Stack 2, Pantry, and what was previously known as custom snapshots.
```
Comment (alexeyzab) [inline posts/new-snapshot-format.md]: Does the new `curator` tool have a repo? Might be worth including the link to it, IMO.

**Comment by alexeyzab** [comment]
Looks good! :+1:

File: posts/new-snapshot-format.md (hunk 2 above)
Comment (snoyberg) [reply posts/new-snapshot-format.md]: Doh, good catch! It's in the Stack repo, I'll link to the subdir.

**Comment by snoyberg** [comment]
Thanks for the reviews! Merging.

**Pull request merged**